lib/galaxy/tools/toolbox/base.py

```diff
@@ -1073,9 +1078,9 @@ def to_dict(self, trans, in_panel=True, **kwds):
             for elt in panel_elts:
                 # Only use cache for objects that are Tools.
                 if hasattr(elt, "tool_type"):
-                    rval.append(self.get_tool_to_dict(trans, elt))
+                    rval.append(self.get_tool_to_dict(trans, elt, tool_help=tool_help))
                 else:
-                    kwargs = dict(trans=trans, link_details=True, toolbox=self)
+                    kwargs = dict(trans=trans, link_details=True, tool_help=tool_help, toolbox=self)
                     rval.append(elt.to_dict(**kwargs))
         else:
             filter_method = self._build_filter_method(trans)
```

Comment:
There's another `get_tool_to_dict()` call at line 1091 below where `, tool_help=tool_help` should also probably be added.

Reply:
thanks! fixed